crates/rethnet_rpc_server/src/lib.rs

```diff
@@ -44,6 +44,7 @@ type RethnetStateType = Arc<RwLock<Box<dyn SyncState<StateError>>>>;
 struct AppState {
     rethnet_state: RethnetStateType,
     fork_client: Option<Arc<Mutex<RpcClient>>>,
+    fork_block_number: Option<usize>,
```

Comment:
Is a `U256` not supported?

Reply:
I modeled this after [the existing `HardhatNetworkForkingUserConfig`](https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-core/src/types/config.ts#L91), which specifies it as a `number` rather than a `bigint`.

Reply:
In the blockchain it's a `U256`, so I'd perform an "upgrade" here and just use that type to support larger numbers too